### PR TITLE
Minor Frontend Fixes

### PR DESCRIPTION
**Pull request by FinalAngel**
- Set correct z-index for messages so it appears underneath the toolbar
- Cutting a plugin causes an error as Message.open provides incorrect params
- Restore default ``.cms {}`` css as this causes the tooltip to be underneath certain content elements with ``position: relative``.

### CODE REVIEW

File: cms/static/cms/sass/components/_toolbar.scss [hunk 1]
```diff
@@ -2,13 +2,6 @@
 // TOOLBAR
 
 .cms-toolbar {
-    position: absolute;
-    top: $toolbar-debug-height;
-    left: 0;
-    z-index: z(base);
-    width: 100%;
-    margin-top: -$toolbar-height - 10px;
```
Comment (vxsx) [inline cms/static/cms/sass/components/_toolbar.scss]: i think margin top was actually needed here

Comment (FinalAngel) [inline cms/static/cms/sass/components/_toolbar.scss]: that was actually not in original code

Comment (FinalAngel) [inline cms/static/cms/sass/components/_toolbar.scss]: restored

**Comment by vxsx** [comment]
:+1:

**Comment by mkoistinen** [comment]
:+1:

**Pull request merged**